resources/zib/zib-MedicalDevice.Product.xml

```diff
@@ -0,0 +1,179 @@
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Device.identifier:productID">
+      <path value="Device.identifier" />
+      <sliceName value="productID" />
+      <short value="ProductID" />
+      <definition value="Unique identification of the product, such as the serial number. The `identifier` may only be used if it also identifies the resource. For this the serial number needs to be unique to one device.&#xD;&#xA;&#xD;&#xA;The serial number which is a component of the production identifier (PI), a conditional, variable portion of a UDI. The identifier.type code should be set to “SNO”(Serial Number) and the system left empty." />
+      <alias value="ProductID" />
+      <max value="1" />
+      <patternIdentifier>
+        <type>
+          <coding>
+            <system value="http://terminology.hl7.org/CodeSystem/v2-0203" />
+            <code value="SNO" />
+          </coding>
+        </type>
+      </patternIdentifier>
+      <mapping>
+        <identity value="zib-medicaldevice-v3.3.1-2020EN" />
+        <map value="NL-CM:10.1.4" />
+        <comment value="ProductID" />
+      </mapping>
+    </element>
+    <element id="Device.udiCarrier">
+      <path value="Device.udiCarrier" />
+      <max value="1" />
+    </element>
+    <element id="Device.udiCarrier.deviceIdentifier">
+      <path value="Device.udiCarrier.deviceIdentifier" />
+      <short value="ProductID" />
+      <definition value="The device identifier (DI) is a mandatory, fixed portion of a UDI that identifies the labeler and the specific version or model of a device.&#xD;&#xA;This represents also the GS1's Global Trade Item Number (GTIN)" />
+      <alias value="ProductID" />
+      <mapping>
+        <identity value="zib-medicaldevice-v3.3.1-2020EN" />
+        <map value="NL-CM:10.1.4" />
+        <comment value="ProductID" />
+      </mapping>
+    </element>
+    <element id="Device.udiCarrier.issuer">
+      <path value="Device.udiCarrier.issuer" />
+      <short value="ProductID" />
+      <definition value="Organization that is charged with issuing UDIs for devices. For the zib, this would be the Global Trade Item Number (GTIN) or &#xD;&#xA;Health Industry Bar Code (HIBC). &#xD;&#xA;The are https://www.gs1.org/gtin and http://hl7.org/fhir/NamingSystem/hibcc, respectively." />
+      <alias value="ProductID" />
+      <mapping>
+        <identity value="zib-medicaldevice-v3.3.1-2020EN" />
+        <map value="NL-CM:10.1.4" />
+        <comment value="ProductID" />
+      </mapping>
+    </element>
+    <element id="Device.udiCarrier.carrierHRF">
+      <path value="Device.udiCarrier.carrierHRF" />
+      <short value="ProductID" />
+      <alias value="ProductID" />
+      <mapping>
+        <identity value="zib-medicaldevice-v3.3.1-2020EN" />
+        <map value="NL-CM:10.1.4" />
+        <comment value="ProductID" />
+      </mapping>
+    </element>
+    <element id="Device.manufactureDate">

```

Comment:
Should this element be mapped to NL-CM:10.1.4 as well (compare with expirationDate)?